assets/contracts.json

```diff
@@ -1 +1 @@
-{"posts":"0x01A0bD87E68a3e2F1a6ef161679241756fa4DaBD","comment_store":"0xca36423723778BC799641738e2C17a0ad8DAd18C"}
\ No newline at end of file
+{"posts":"0x302cFcFC1e667a78f6f2E06AA8a36255a9BEF914","comment_store":"0x557f1271443824ECE86aC1a5409cFF00A2d15D55"}
```

Comment:
You can add this to `.gitignore` I believe.

Reply:
It's already done in one of the PRs.